20220610.md

```diff
@@ -45,3 +45,124 @@ This last fortnight, over the 17 pull requests or activities reported in the ope
+
+There are no shortage of calculator apps for Sailfish OS, and for more complex operations you can always drop to a Python console. But while not being a fully-fledged programming language, Babbage provides a good selection of more advanced functions and an interface that is much easier to use than the console, managing to maintain a careful balance between usability and functionality.
+
+The latest update to Babbage is avilable on the Jolla Store, and for aarch64 users for the first time.
+
+## SailBabel
+
+Another nice app form Heiko Bauke (**rabauke**), once installed SailBabel will declare itself as simply "Dictionary" in your app drawer. A clean install will present a list of directories rather than a list of words: it's up to you to provide a word list, which can either be a list of translations or a list of definitions, so this can act as either type of dictionary. The *de-facto* choice for translation lists being [dict.cc](http://www1.dict.cc) with its rather pecular email-based [download process](http://www1.dict.cc/translation_file_request.php?l=e). The wordlists come as text files which you can load directly into SailBabel, after which you can search the lists to display the results in a clean way, using different highlighting to distinguish between the search result and the translation or definition. The app also provides an easily accessible search history so you can skip forwards and backwards through your search terms.
+
+The obvious competitor to SailBabel is Sebastian Wolf's (**WerkWolf**) Wunderfitz. While Wunderfitz has more features, with its scan-translations, toolbar and in-built dictionary list, there's an elegant simplicity to SailBabel that also makes it feel more responsive and lighter to use.
+
+When it comes to either app, the experience is ultimately tied to the quality of the database in use. I found translating between Finnish and English using the [dict.cc](http://www1.dict.cc) database to be a bit hit-and-miss. Often the stem or *vartalo* word exists is in the database, but because neither app supports fuzzy search, getting at it wasn't always obvious.
```

Comment:
Maybe myself, but the sentence "Often the stem […] exists is in the database". Maybe a typo with the two verbs, or I'm missing the sentence construction.

Reply:
You're right, the sentence was wrong. I've reworked it a bit and hopefully it's clearer now.